py3-lxml.yaml

```diff
@@ -0,0 +1,47 @@
+# Generated from https://pypi.org/project/lxml/
+package:
+  name: py3-lxml
+  version: 4.9.3
+  epoch: 0
+  description: Powerful and Pythonic XML processing library combining libxml2/libxslt with the ElementTree API.
+  copyright:
+    - license: BSD-3-Clause
+  dependencies:
+    runtime:
+      - python3
+
+environment:
+  contents:
+    packages:
+      - ca-certificates-bundle
+      - wolfi-base
+      - busybox
+      - build-base
+      - python3-dev
+      - py3-setuptools
+      - libxml2-dev
+      - libxslt-dev
+      - gcc
+      - cython<3.0.0 # lxml 4.9.3 is incompatible with cython 3+
```

Comment:
```suggestion
      - cython~0 # lxml 4.9.3 is incompatible with cython 3+
```

Reply:
Does that still apply if the package is called `cython-0`?

Reply:
@kaniini updated! GitHub didn't allow me to accept your suggestion because I force-pushed 😞